packages/contracts/contracts/multisig/MultiSigWallet.sol

```diff
@@ -78,15 +78,15 @@ contract MultiSigWallet {
     _;
   }
 
-  modifier notNull(address _address) {
-    require(_address != 0);
+  modifier notNull(address checkedAddress) {
```

Comment:
`checkedAddress` and `checkedRequired` below both sound a bit awkward to me. Could `checkRequired` be `requiredConfirmationsProposed`? Or something?